frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx

```diff
@@ -0,0 +1,152 @@
+    const columns: ColumnsType<FlattenedFunnelStep> = []
+
+    columns.push({
+        title: '',
+        render: function RenderCheckboxOrGlyph({}, step: FlattenedFunnelStep): JSX.Element {
+            if (step.breakdownIndex === undefined) {
+                // Not a breakdown value; show a step-order glyph
+                return <SeriesGlyph variant="funnel-step-glyph">{humanizeOrder(step.order)}</SeriesGlyph>
+            }
+            const isVisible = !!(step.breakdown && visibilityMap[step.breakdown])
+            const color = getStepColor(step, !!filters.breakdown)
+            return (
+                <div style={{ marginLeft: '0.15em' }}>
+                    <PHCheckbox
+                        color={color}
+                        checked={isVisible}
+                        onChange={() => step.breakdown && setVisibility(step.breakdown, !isVisible)}
+                    />
+                </div>
+            )
+        },
+        fixed: 'left',
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Step',
+        render: function RenderLabel({}, step: FlattenedFunnelStep): JSX.Element {
+            const isVisible = !!(step.breakdown && visibilityMap[step.breakdown])
+            const isBreakdownChild = !!filters.breakdown && !step.isBreakdownParent
+            const color = getStepColor(step, !!filters.breakdown)
+            return (
+                <SeriesToggleWrapper
+                    id={step.order}
+                    toggleVisibility={() => step.breakdown && setVisibility(step.breakdown, !isVisible)}
+                    style={{ display: 'flex', alignItems: 'center' }}
+                >
+                    {isBreakdownChild ? (
+                        <div style={{ maxWidth: 270, wordBreak: 'break-word' }}>
+                            {formatBreakdownLabel(step.breakdown, cohorts)}
+                        </div>
+                    ) : (
+                        <div style={{ flexGrow: 1 }}>
+                            <InsightLabel
+                                seriesColor={color}
+                                fallbackName={step.name}
+                                hasMultipleSeries={steps.length > 1}
+                                breakdownValue={step.breakdown}
+                                hideBreakdown
+                                hideIcon
+                            />
+                        </div>
+                    )}
+                </SeriesToggleWrapper>
+            )
+        },
+        fixed: 'left',
+        width: 300,
+    })
+
+    columns.push({
+        title: 'Completed',
+        render: function RenderCompleted({}, step: FlattenedFunnelStep): JSX.Element {
+            return (
+                <ValueInspectorButton onClick={() => openPersonsModal(step, step.count + 1, step.breakdown_value)}>
+                    {step.count}
+                </ValueInspectorButton>
+            )
+        },
+        width: 30,
+    })
+
+    columns.push({
```

Comment:
Maybe more intuitive to have the conversion rate after the dropped off number?

Reply:
Not sure what you mean? Right now it's `value, percentage, value, percentage` and conversion before dropoff,  just as it is on the main funnel graph

Reply:
I think you're right, I thought it was, # users completed, # users dropped off, % conversion rate, which is what feels most intuitive.